site/public/js/server.js

```diff
@@ -1096,6 +1096,12 @@ function enableTabsInTextArea(jQuerySelector) {
     });
 }
 
+function confirmBypass(str, redirect){
+    confirmed = confirm(str);
+    if (confirmed)
```

Comment:
`confirmed` needs to be properly declared with `let` or `const`.  Alternatively, you could just move `confirm(str)` to the body of the if-statement.